### PR TITLE
AG-12510 - Misc cleanup post node matrix removal.

### PR DESCRIPTION
**Pull request by alantreadway**
https://ag-grid.atlassian.net/browse/AG-12510

### CODE REVIEW

File: packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts [hunk 1]
```diff
@@ -28,19 +26,10 @@ export abstract class AnnotationScene extends _Scene.Group {
...
         return _Scene.Transformable.toCanvas(
             this,
             _Scene.Group.computeChildrenBBox(this.children.filter((node) => !(node instanceof Handle)))
         );
     }
-
-    override render(renderCtx: _Scene.RenderContext) {
-        super.render(renderCtx);
-        this.cachedBBoxWithoutHandles = this.computeBBoxWithoutHandles();
-    }
```
Comment (lsjroberts) [inline packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts:45]: Just to double check, this was added to stop the anchor point (and therefore the floating options buttons) from shifting after you first select an annotation and then the next render changed the bbox.

**Review by lsjroberts — APPROVED** [review]
LGTM

File: packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts (hunk 1 above)
Comment (alantreadway) [reply packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts:45]: I'll double check the use-case, but this code was not being used so I don't believe it is currently having any effect.

Comment (alantreadway) [reply packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts:45]: Specifically, `getCachedBBoxWithoutHandles()` is not called anywhere, so the caching is not being used?

File: packages/ag-charts-community/benchmarks/__image_snapshots__/large-dataset-test-ts-large-dataset-benchmark-after-load-1-x-datum-highlight-1-snap.png
Comment (alantreadway) [inline packages/ag-charts-community/benchmarks/__image_snapshots__/large-dataset-test-ts-large-dataset-benchmark-after-load-1-x-datum-highlight-1-snap.png:1]: Reverts snapshot changes from #2245 as the benchmarks have been updated to use `Transform.toCanvasPoint()`.

File: packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts (hunk 1 above)
Comment (alantreadway) [reply packages/ag-charts-enterprise/src/features/annotations/scenes/annotationScene.ts:45]: Discussed offline - agreed this code was redundant.

**Pull request merged**